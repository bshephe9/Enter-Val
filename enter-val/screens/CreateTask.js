import React from 'react';
import { ScrollView, View, Button } from 'react-native';
import { FormLabel, FormInput } from 'react-native-elements';
import firebase from 'firebase';

class CreateTask extends React.Component {
    state = {
        startTime: '',
        endTime: '',
        task: '',
        body: '',
        id: 0
    }

    submit = () => {
        const startTime = this.state.startTime;
        const endTime = this.state.endTime;
        const task = this.state.task;
        const body = this.state.body;
        //Update maybe using the length of the array from array 
        const id = this.state.id + 1;
        const newTask = {
            startTime: startTime,
            endTime: endTime,
            task: task,
            body: body,
            // id: id
        }
        this.another(newTask);
    }

    another = i => {
        firebase.database().ref().push(i); //pushing to the database
        //Emptying the form 
        this.setState({
            startTime: '',
            endTime: '',
            task: '',
            body: ''
        }); //TODO find how to increase the counter to each card has it own id
        this.props.navigation.navigate('UserScreen')
    }

    introTime = event => {
        this.setState({ startTime: event });
    };

    endTime = event => {
        this.setState({ endTime: event });
    };

    changeBody = event => {
        this.setState({ body: event });
    };

    changeTask = event => {
        this.setState({ task: event });
    };

    render() {
        return (

            <ScrollView>
                <View>
                    <FormLabel>Start Time</FormLabel>
                    <FormInput onChangeText={this.introTime}></FormInput>
                </View>
                <View>
                    <FormLabel>End Time</FormLabel>
                    <FormInput onChangeText={this.endTime}></FormInput>
                </View>
                <View>
                    <FormLabel>Task Title</FormLabel>
                    <FormInput value={this.state.task} onChangeText={this.changeTask}></FormInput>
                </View>
                <View>
                    <FormLabel>Body</FormLabel>
                    <FormInput value={this.state.body} onChangeText={this.changeBody}></FormInput>
                    <Button title="Submit" onPress={this.submit} />
                </View>
            </ScrollView>

        );
    }
}

export default CreateTask;