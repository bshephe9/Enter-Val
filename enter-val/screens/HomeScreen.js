import React from 'react';
import {
  StyleSheet,
  Text,
  View,
  ImageBackground,
} from 'react-native';
import Button from 'react-native-button';



export default class HomeScreen extends React.Component {

  state = {
    curTime: null,
    curDate: null
  }

  componentDidMount() {
    setInterval(() => {
      this.setState({
        curTime: new Date().toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' }),
        curDate: new Date().toLocaleDateString("en-US", { weekday: "long", month: "short", day: "numeric" })
      })
    }, 1000)
  }

  render() {
    return (
      <ImageBackground
        source={require('../assets/images/bg.jpg')}
        style={styles.container}>

        <View style={styles.overlayContainer}>
          <View>
            <Text style={styles.timer}>{this.state.curDate}</Text>
          </View>
          <View>
            <Text style={styles.header}>enterVal</Text>
          </View>

          <View>
            <Text style={styles.timer}>{this.state.curTime}</Text>
          </View>
          <View>
            <Button
              style={{ fontSize: 20, color: '#fff', marginTop: 255, borderColor: '#fa8072', backgroundColor: '#fa8072', padding: 4, paddingLeft: 20, paddingRight: 20, borderRadius: 5, overflow: 'hidden' }}
              styleDisabled={{ color: 'red' }}
              onPress={() => this.doSomething()}>
              get started
            </Button>
          </View>
        </View>

      </ImageBackground>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    width: '100%',
    height: '100%',
  },
  overlayContainer: {
    alignItems: 'center',
    color: 'black',
    marginTop: 120,
  },
  header: {
    fontSize: 35,
    borderColor: '#fff',
    fontWeight: 'bold',
    color: '#fff',
    borderWidth: 2,
    padding: 20,
    paddingLeft: 40,
    paddingRight: 40,
  },
  timer: {
    color: '#fff',
    fontSize: 25,
    marginTop: 20,
  }
});