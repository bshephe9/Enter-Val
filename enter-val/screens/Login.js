// Login.js
import React from 'react'
import { StyleSheet, Text, TextInput, View, Button } from 'react-native'
import firebase from 'firebase'
export default class Login extends React.Component {
  state = { email: '', password: '', errorMessage: null }
  handleLogin = () => {
      const { email, password } = this.state
      firebase
        .auth()
        .signInWithEmailAndPassword(email, password)
        .then(() => this.props.navigation.navigate('Main'))
        .catch(error => this.setState({ errorMessage: error.message}))
    // TODO: Firebase stuff...
    console.log('handleLogin')
  }