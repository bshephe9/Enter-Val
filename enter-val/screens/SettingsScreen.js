//!! this is actually the log in page so design it as so :)
import React from 'react';
import {
  StyleSheet,
  View,
} from 'react-native';
import Button from 'react-native-button';



export default class SettingsScreen extends React.Component {
  render() {
    return (
      <View>
        <Button
          style={{ fontSize: 20, color: '#fff', marginTop: 400, marginLeft: 125, borderColor: '#fa8072', backgroundColor: '#fa8072', padding: 4, paddingLeft: 20, paddingRight: 20, borderRadius: 5, width: 150, overflow: 'hidden' }}
          styleDisabled={{ color: 'red' }}
          onPress={() => this.props.navigation.navigate('UserScreen')}>
          Log in
            </Button>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    width: '100%',
    height: '100%',
  },
  content: {
    fontSize: 15,
    alignItems: 'center',
  }
});
