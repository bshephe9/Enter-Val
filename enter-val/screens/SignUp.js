// SignUp.js
import React from 'react';
import { StyleSheet, Text, TextInput, View, TouchableOpacity, ImageBackground } from 'react-native';
import firebase from 'firebase';

export default class SignUp extends React.Component {
  state = { email: '', password: '', errorMessage: null }
  handleSignUp = () => {
    firebase
      .auth()
      .createUserWithEmailAndPassword(this.state.email,
        this.state.password)
      .then(() => this.props.navigation.navigate('UserScreen'))
      .catch(error => this.setState({ errorMessage: error.message }))
    // TODO: Firebase stuff...
    console.log('handleSignUp')
  }
  render() {
    return (
      <ImageBackground style={{
        flex: 1,
        resizeMode: 'cover'
      }}
        source={require('../assets/images/bg.jpg')}
        blurRadius={5}>

        <View style={styles.container}>
          <Text style={{ fontSize: 30, fontWeight: 'bold', marginBottom: 20, color: '#fff' }}>
            Get Started with EnterVal</Text>

          {this.state.errorMessage &&
            <Text style={{ color: 'red' }}>
              {this.state.errorMessage}
            </Text>}
          <TextInput
            placeholder="Email"
            placeholderTextColor="black"
            autoCapitalize="none"
            style={styles.textInput}
            onChangeText={email => this.setState({ email })}
            value={this.state.email}
          />
          <TextInput
            secureTextEntry
            placeholder="Password"
            placeholderTextColor="black"
            autoCapitalize="none"
            style={styles.textInput}
            onChangeText={password => this.setState({ password })}
            value={this.state.password}
          />
          <TouchableOpacity style={styles.signup} title="Sign Up" onPress={this.handleSignUp} />
          <TouchableOpacity style={styles.login}
            title="Already have an account? Login"
            onPress={() => this.props.navigation.navigate('Login')}
          />
        </View>
      </ImageBackground >
    )
  }
}
const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center',
    marginTop: 200,
    color: '#fff',
    borderRadius: 5,
  },
  textInput: {
    height: 40,
    width: '90%',
    borderColor: '#fff',
    backgroundColor: '#fff',
    borderWidth: 1,
    marginTop: 8,
    color: 'black',
    borderRadius: 5,
  },
  signup: {
    height: 35,
    width: '35%',
    borderColor: '#e9967a',
    backgroundColor: '#e9967a',
    borderWidth: 1,
    marginTop: 8,
    borderRadius: 5,
  },
  login: {
    height: 35,
    width: '35%',
    borderColor: '#e9967a',
    backgroundColor: '#e9967a',
    borderWidth: 1,
    marginTop: 8,
    color: '#fff',
    borderRadius: 5,
  },
})
