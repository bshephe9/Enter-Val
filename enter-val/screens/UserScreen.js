import React from 'react';
import {
  ScrollView,
  Text,
  View,
} from 'react-native';
import {
  Card
} from 'react-native-elements';
import { FloatingAction } from 'react-native-floating-action';



  export default class UserScreen extends React.Component {
    render() {
      return (

      <ScrollView style={{ paddingVertical: 20 }}>

        <Card title='8:00 - 11:00 AM'>
          <View
            style={{
              backgroundColor: '#bcbec1',
              alignItems: 'center',
              justifyContent: 'center',
              width: 80,
              height: 80,
              borderRadius: 40,
              alignSelf: 'center',
              marginBottom: 20
            }}
          >
            <Text style={{ color: 'white', fontSize: 28 }}>JD</Text>
          </View>
        </Card>
        <Card title='11:00 - 2:00 PM'>
          <View
            style={{
              backgroundColor: '#bcbec1',
              alignItems: 'center',
              justifyContent: 'center',
              width: 80,
              height: 80,
              borderRadius: 40,
              alignSelf: 'center',
              marginBottom: 20
            }}
          >
            <Text style={{ color: 'white', fontSize: 28 }}>JD</Text>
          </View>
        </Card>

        <Card title='2:00 - 6:00 PM'>
          <View
            style={{
              backgroundColor: '#bcbec1',
              alignItems: 'center',
              justifyContent: 'center',
              width: 80,
              height: 80,
              borderRadius: 40,
              alignSelf: 'center',
              marginBottom: 20
            }}
          >
            <Text style={{ color: 'white', fontSize: 28 }}>JD</Text>
          </View>
        </Card>

        <Card title='6:00 - 10:00 PM'>
          <View
            style={{
              backgroundColor: '#bcbec1',
              alignItems: 'center',
              justifyContent: 'center',
              width: 80,
              height: 80,
              borderRadius: 40,
              alignSelf: 'center',
              marginBottom: 20
            }}
          >
            <Text style={{ color: 'white', fontSize: 28 }}>JD</Text>
          </View>
        </Card>

        <View>
          <FloatingAction 
            actions={actions}
            onPressItem={
              (name) => {
                console.log(`selected button: ${name}`);
              }
            }
          />
        </View>

      </ScrollView>
    );
  }

const actions = [{
  text: 'Home',
  name: 'bt_home',
  position: 1
}, {
  text: 'Tasks',
  name: 'bt_task',
  position: 2
}, {
  text: 'Notes',
  name: 'bt_notes',
  position: 3
}, {
  text: 'Log Out',
  name: 'bt_logout',
  position: 4
}];

