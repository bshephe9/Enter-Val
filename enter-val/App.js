import React from 'react';
import { StyleSheet, Text, View } from 'react-native';
import {
  createStackNavigator,
  createAppContainer,
} from 'react-navigation'

//!! this is where we import new "screens"
import HomeScreen from './screens/HomeScreen';
import SettingsScreen from './screens/SettingsScreen'
import UserScreen from './screens/UserScreen';
import CreateTask from './screens/CreateTask';

//!! this renders new screens for them to be used in the app
const AppNavigator = createStackNavigator({
  HomeScreen: { screen: HomeScreen },
  SettingsScreen: { screen: SettingsScreen },
  UserScreen: { screen: UserScreen },
  CreateTask: { screen: CreateTask }
});


//!! this container was created during V3 and is required for the routes to work
const App = createAppContainer(AppNavigator);


//!! it also needs to be exported
export default App;