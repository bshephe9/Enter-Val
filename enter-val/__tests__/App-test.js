import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';
import HomeScreen from '../screens/HomeScreen';
import CreateTask from "../screens/CreateTask";
import Login from "../screens/Login";
import SignUp from "../screens/SignUp";
import UserScreen from "../screens/UserScreen";
//TODO check if we need this
import SettingsScreen from "../screens/SettingsScreen";

//* testing App

it('App renders without crashing', () => {
const rendered = renderer.create(<App />).toJSON();
expect(rendered).toBeTruthy();
});

it('App test against snapshot', () => {
const tree = renderer.create(<App />).toJSON();
expect(tree).toMatchSnapshot();
});

//* testing HomeScreen

it('Home Screen renders without crashing', () => {
    const rendered = renderer.create(<HomeScreen />).toJSON();
    expect(rendered).toBeTruthy();
    });
    
    it('Home Screen test against snapshot', () => {
    const tree = renderer.create(<HomeScreen />).toJSON();
    expect(tree).toMatchSnapshot();
    });

//* testing Login

it('Login renders without crashing', () => {
    const rendered = renderer.create(<Login />).toJSON();
    expect(rendered).toBeTruthy();
    });
    
    it('Login test against snapshot', () => {
    const tree = renderer.create(<Login />).toJSON();
    expect(tree).toMatchSnapshot();
    });    

//* testing Sign Up

it('Sign up renders without crashing', () => {
    const rendered = renderer.create(<SignUp />).toJSON();
    expect(rendered).toBeTruthy();
    });
    
    it('Sign up test against snapshot', () => {
    const tree = renderer.create(<SignUp />).toJSON();
    expect(tree).toMatchSnapshot();
    });

//* testing User Screen

it('User Screen renders without crashing', () => {
    const rendered = renderer.create(<UserScreen />).toJSON();
    expect(rendered).toBeTruthy();
    });
    
    it('User Screen test against snapshot', () => {
    const tree = renderer.create(<UserScreen />).toJSON();
    expect(tree).toMatchSnapshot();
    });

//* testing Tasks

it('Tasks Page renders without crashing', () => {
    const rendered = renderer.create(<CreateTask />).toJSON();
    expect(rendered).toBeTruthy();
    });
    
    it('Tasks Page test against snapshot', () => {
    const tree = renderer.create(<CreateTask />).toJSON();
    expect(tree).toMatchSnapshot();
    });

//TODO testing SettingsScreen?? 

it('Settings renders without crashing', () => {
    const rendered = renderer.create(<SettingsScreen />).toJSON();
    expect(rendered).toBeTruthy();
    });
    
    it('Settings test against snapshot', () => {
    const tree = renderer.create(<SettingsScreen />).toJSON();
    expect(tree).toMatchSnapshot();
    });