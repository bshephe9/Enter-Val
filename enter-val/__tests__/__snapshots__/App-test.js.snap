// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test against snapshot 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    enabled={false}
    hitSlop={
      Object {
        "right": 50,
      }
    }
    maxDeltaY={20}
    minOffsetX={5}
    onGestureEvent={[Function]}
    onHandlerStateChange={[Function]}
  >
    <View
      style={
        Object {
          "backgroundColor": "#eee",
          "flex": 1,
          "flexDirection": "column-reverse",
          "overflow": "hidden",
        }
      }
    >
      <View
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <View
          pointerEvents="auto"
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "opacity": 1,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "transform": Array [
                Object {
                  "translateX": 0,
                },
                Object {
                  "translateY": 0,
                },
              ],
            }
          }
        >
          <View
            accessibilityElementsHidden={false}
            style={
              Object {
                "backgroundColor": "#fff",
                "flex": 1,
              }
            }
          >
            <View
              accessibilityIgnoresInvertColors={true}
              style={
                Object {
                  "flex": 1,
                  "resizeMode": "cover",
                }
              }
            >
              <Image
                source={1}
                style={
                  Array [
                    Object {
                      "bottom": 0,
                      "left": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                    },
                    Object {
                      "height": undefined,
                      "width": undefined,
                    },
                    undefined,
                  ]
                }
              />
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "color": "black",
                    "marginTop": 25,
                  }
                }
              >
                <View>
                  <Text
                    style={
                      Object {
                        "color": "#fff",
                        "fontSize": 25,
                        "marginBottom": 6,
                        "marginTop": 30,
                      }
                    }
                  />
                </View>
                <View>
                  <Text
                    style={
                      Object {
                        "borderColor": "#fff",
                        "borderWidth": 2,
                        "color": "#fff",
                        "fontSize": 35,
                        "fontWeight": "bold",
                        "padding": 20,
                        "paddingLeft": 40,
                        "paddingRight": 40,
                      }
                    }
                  >
                    enterVal
                  </Text>
                </View>
                <View>
                  <Text
                    style={
                      Object {
                        "color": "#fff",
                        "fontSize": 25,
                        "marginBottom": 6,
                        "marginTop": 30,
                      }
                    }
                  />
                </View>
                <View>
                  <Text
                    style={
                      Object {
                        "color": "#fff",
                        "fontSize": 25,
                        "marginBottom": 6,
                        "marginTop": 30,
                      }
                    }
                  >
                    0
                    ยบ
                  </Text>
                </View>
                <View>
                  <View
                    accessible={true}
                    isTVSelectable={true}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "opacity": 1,
                      }
                    }
                  >
                    <Text
                      style={
                        Array [
                          Object {
                            "color": "#007aff",
                            "fontSize": 17,
                            "fontWeight": "500",
                            "textAlign": "center",
                          },
                          null,
                          Object {
                            "backgroundColor": "#e9967a",
                            "borderColor": "#fa8072",
                            "borderRadius": 5,
                            "color": "#fff",
                            "fontSize": 20,
                            "marginTop": 300,
                            "overflow": "hidden",
                            "padding": 4,
                            "paddingLeft": 20,
                            "paddingRight": 20,
                          },
                          null,
                        ]
                      }
                    >
                      get started
                    </Text>
                  </View>
                </View>
                <View>
                  <Text
                    style={
                      Object {
                        "color": "#fff",
                        "fontSize": 18,
                        "fontStyle": "italic",
                        "marginTop": 190,
                      }
                    }
                  >
                    "Keep going. Be all in."
                  </Text>
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
      <View
        onLayout={[Function]}
        pointerEvents="box-none"
        style={
          Object {
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#FFF",
              "transform": Array [
                Object {
                  "translateX": -750,
                },
              ],
            }
          }
        >
          <View
            onLayout={[Function]}
            pointerEvents="box-none"
            style={
              Object {
                "backgroundColor": "#FFF",
                "borderBottomColor": "#A7A7AA",
                "borderBottomWidth": 0.5,
                "height": 64,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 20,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`Home Screen test against snapshot 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "flex": 1,
      "resizeMode": "cover",
    }
  }
>
  <Image
    source={1}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "color": "black",
        "marginTop": 25,
      }
    }
  >
    <View>
      <Text
        style={
          Object {
            "color": "#fff",
            "fontSize": 25,
            "marginBottom": 6,
            "marginTop": 30,
          }
        }
      />
    </View>
    <View>
      <Text
        style={
          Object {
            "borderColor": "#fff",
            "borderWidth": 2,
            "color": "#fff",
            "fontSize": 35,
            "fontWeight": "bold",
            "padding": 20,
            "paddingLeft": 40,
            "paddingRight": 40,
          }
        }
      >
        enterVal
      </Text>
    </View>
    <View>
      <Text
        style={
          Object {
            "color": "#fff",
            "fontSize": 25,
            "marginBottom": 6,
            "marginTop": 30,
          }
        }
      />
    </View>
    <View>
      <Text
        style={
          Object {
            "color": "#fff",
            "fontSize": 25,
            "marginBottom": 6,
            "marginTop": 30,
          }
        }
      >
        0
        ยบ
      </Text>
    </View>
    <View>
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007aff",
                "fontSize": 17,
                "fontWeight": "500",
                "textAlign": "center",
              },
              null,
              Object {
                "backgroundColor": "#e9967a",
                "borderColor": "#fa8072",
                "borderRadius": 5,
                "color": "#fff",
                "fontSize": 20,
                "marginTop": 300,
                "overflow": "hidden",
                "padding": 4,
                "paddingLeft": 20,
                "paddingRight": 20,
              },
              null,
            ]
          }
        >
          get started
        </Text>
      </View>
    </View>
    <View>
      <Text
        style={
          Object {
            "color": "#fff",
            "fontSize": 18,
            "fontStyle": "italic",
            "marginTop": 190,
          }
        }
      >
        "Keep going. Be all in."
      </Text>
    </View>
  </View>
</View>
`;

exports[`Login test against snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Login
  </Text>
  <TextInput
    allowFontScaling={true}
    autoCapitalize="none"
    onChangeText={[Function]}
    placeholder="Email"
    style={
      Object {
        "borderColor": "gray",
        "borderWidth": 1,
        "height": 40,
        "marginTop": 8,
        "width": "90%",
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <TextInput
    allowFontScaling={true}
    autoCapitalize="none"
    onChangeText={[Function]}
    placeholder="Password"
    secureTextEntry={true}
    style={
      Object {
        "borderColor": "gray",
        "borderWidth": 1,
        "height": 40,
        "marginTop": 8,
        "width": "90%",
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <View
    accessibilityRole="button"
    accessibilityStates={Array []}
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Login
      </Text>
    </View>
  </View>
  <View
    accessibilityRole="button"
    accessibilityStates={Array []}
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Don't have an account? Sign Up
      </Text>
    </View>
  </View>
</View>
`;

exports[`Settings test against snapshot 1`] = `
<View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#007aff",
            "fontSize": 17,
            "fontWeight": "500",
            "textAlign": "center",
          },
          null,
          Object {
            "backgroundColor": "#fa8072",
            "borderColor": "#fa8072",
            "borderRadius": 5,
            "color": "#fff",
            "fontSize": 20,
            "marginLeft": 100,
            "marginRight": 100,
            "marginTop": 700,
            "overflow": "hidden",
            "padding": 4,
            "paddingLeft": 5,
            "paddingRight": 5,
          },
          null,
        ]
      }
    >
      Log in
    </Text>
  </View>
</View>
`;

exports[`Sign up test against snapshot 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "flex": 1,
      "resizeMode": "cover",
    }
  }
>
  <Image
    blurRadius={5}
    source={1}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "borderRadius": 5,
        "color": "#fff",
        "flex": 1,
        "marginTop": 200,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#fff",
          "fontSize": 30,
          "fontWeight": "bold",
          "marginBottom": 20,
        }
      }
    >
      Get Started with EnterVal
    </Text>
    <TextInput
      allowFontScaling={true}
      autoCapitalize="none"
      onChangeText={[Function]}
      placeholder="Email"
      placeholderTextColor="black"
      style={
        Object {
          "backgroundColor": "#fff",
          "borderColor": "#fff",
          "borderRadius": 5,
          "borderWidth": 1,
          "color": "black",
          "height": 40,
          "marginTop": 8,
          "width": "90%",
        }
      }
      underlineColorAndroid="transparent"
      value=""
    />
    <TextInput
      allowFontScaling={true}
      autoCapitalize="none"
      onChangeText={[Function]}
      placeholder="Password"
      placeholderTextColor="black"
      secureTextEntry={true}
      style={
        Object {
          "backgroundColor": "#fff",
          "borderColor": "#fff",
          "borderRadius": 5,
          "borderWidth": 1,
          "color": "black",
          "height": 40,
          "marginTop": 8,
          "width": "90%",
        }
      }
      underlineColorAndroid="transparent"
      value=""
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#e9967a",
          "borderColor": "#e9967a",
          "borderRadius": 5,
          "borderWidth": 1,
          "height": 35,
          "marginTop": 8,
          "opacity": 1,
          "width": "35%",
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#e9967a",
          "borderColor": "#e9967a",
          "borderRadius": 5,
          "borderWidth": 1,
          "color": "#fff",
          "height": 35,
          "marginTop": 8,
          "opacity": 1,
          "width": "35%",
        }
      }
    />
  </View>
</View>
`;

exports[`Tasks Page test against snapshot 1`] = `null`;

exports[`User Screen test against snapshot 1`] = `null`;
